src/frontend/src/planner/relation.rs

```diff
@@ -84,6 +87,42 @@ impl Planner {
         }
     }
 
+    pub(super) fn plan_generate_series_function(
+        &mut self,
+        table_function: BoundGenerateSeriesFunction,
+    ) -> Result<PlanRef> {
+        let schema = Schema::new(vec![Field::with_name(
+            DataType::Timestamp,
+            "generate_series",
+        )]);
+        let args = table_function.args.into_iter();
+
+        let Some((ExprImpl::Literal(start_time), ExprImpl::Literal(stop_time),ExprImpl::Literal(step_interval))) = args.next_tuple() else {
+            return Err(ErrorCode::BindError("Invalid arguments for Generate series function".to_string()).into());
+        };
+        let Some(ScalarImpl::NaiveDateTime(start_time)) = *start_time.get_data() else {
+            return Err(ErrorCode::BindError("Invalid arguments for Generate series function".to_string()).into());
+        };
+
+        let Some(ScalarImpl::NaiveDateTime(stop_time)) = *stop_time.get_data() else {
+            return Err(ErrorCode::BindError("Invalid arguments for Generate series functionn".to_string()).into());
+        };
+
+        let Some(ScalarImpl::Interval(step_interval)) = *step_interval.get_data() else {
+            return Err(ErrorCode::BindError("Invalid arguments for Generate series function".to_string()).into());
+        };
+        dbg!(&start_time);
+        dbg!(&stop_time);
+        dbg!(&step_interval);
+        Ok(LogicalGenerateSeries::create(
```

Comment:
I am confused about whether use NaiveDateTimeWrapper or DataType to store start and stop timestamps. Any Suggestion @TennyZhuang